src/containers/progressBar/ProgressBar.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { render, screen } from "@testing-library/react";
+import { describe, it, expect } from "vitest";
+import ProgressBar from "./ProgressBar";
+import '@testing-library/jest-dom';
+
+/**
+* @vitest-environment jsdom
+*/
+
+describe("Progress Bar",()=> {
+  it("should render the ProgressBarSymbol component", () => {
+    render(<ProgressBar />); // Explicitly pass completed prop
+    const progressBar = screen.getByTestId("progressBar");
+    expect(progressBar).toBeInTheDocument();
+  });
+  it(`should contain correct amount of progress bar symbols based on questionsData prop`, () => {
+    const data = [
+      {
+        current: false,
+        correct: false, 
+        completed: false,
+      },
+      {
+        current: false,
+        correct: false, 
+        completed: false,
+      },      {
+        current: false,
+        correct: false, 
+        completed: false,
```

Comment:
TypeScript mentions, the questionData properties need to be the same as in the type definition, like this:
`{
        isCurrent: false,
        isCorrect: false,
        isCompleted: false,
      },`
      
      Instead of just "current, correct, completed".

Reply:
One of my last commits was changing this parameter to use `is` for clearer understanding and I must have missed it in this test file. Thank you!